doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -594,6 +594,159 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notifications formats
+
+Notifications include two fields: `subscriptionId` (which value is the one of the subscription triggering
+the notification) and `data` (which holds the actual data corresponding to entities and attributes).
+
+The `data` field can use different representation depending on two properties (especified as part of the
+corresponding subscription, see subscriptions operations section for details).
+
+* attrsFormat ("full", "keyValues", "values"; default is "full") to specify how the attributes are
```

Comment:
Whatever the solution would be, it should be based in information in the payload, given that this information should be included in the subscription information retrieved by `GET /v2/subscription` and `GET /v2/subscription/<subId>` operations.

In this sense, I like the idea of "packing" under the same field, but not sure if `options` is the best name for such field, given that we also have `options` as URI param (e.g. used for `count`) and  could be confussing.

Thus, we could use the following:

```
"format": {
  "attrFormat": "keyValues",
  "dataAs": "object"
}
```

being that `format` a subfield of `notification`(I think it is the proper place, as these parameters are defining aspects of the notifications being sent).

What do you think?